components/EditRoomModal/EditRoomModal.js

```diff
@@ -72,22 +67,24 @@ export default function EditRoomModal({ room }) {
       {modalOpen && (
         <StyledModal>
           <StyledOverlay onClick={toggleModal}></StyledOverlay>
+
           <StyledModalContent>
             <StyledModalHeadline>Edit Room</StyledModalHeadline>
-
-            <StyledLabel htmlFor="name">Name</StyledLabel>
-            <StyledInput
-              type="text"
-              id="name"
-              name="title"
-              required
-              value={title}
-              onChange={handleTitleChange}
-            />
-
-            <StyledAddButton type="submit" onClick={handleEdit}>
-              Save edits
-            </StyledAddButton>
+            <form>
+              <StyledLabel htmlFor="name">Name</StyledLabel>
+              <StyledInput
+                type="text"
+                id="name"
+                name="title"
+                required
+                value={title}
+                onChange={handleTitleChange}
+              />
+
+              <StyledAddButton type="submit" onClick={handleEdit}>
+                Save edits
+              </StyledAddButton>
+            </form>
```

Comment:
i'd move the event listener to the form, so that is also triggered if the form is submitted by any other means:

```suggestion
            <form onSubmit={handleEdit}>
              <StyledLabel htmlFor="name">Name</StyledLabel>
              <StyledInput
                type="text"
                id="name"
                name="title"
                required
                value={title}
                onChange={handleTitleChange}
              />

              <StyledAddButton type="submit">
                Save edits
              </StyledAddButton>
            </form>
```

Reply:
I actually forgot to remove the `<form>` since i was just playing around and trying some other methods, so i'll just delete the` <form> `tag :D